src/public/preload.js

```diff
@@ -15,6 +15,29 @@ events.forEach(function (e) {
     });
 });
 
+const userPresenceControl = () => {
+    UserPresence.stopTimer(); //stop userpresence control
+    const AWAY_TIME = 300000; // 5 min
```

Comment:
Yeah any kind of interaction really. Why do you feel like this should be lower? One thing we could do is add a setting, but I don't think we have any kind of "app setting" at the moment, maybe in 3.0